cw/cw305/util/spiflash.py

```diff
@@ -126,8 +126,8 @@ class _Bootstrap:
   _PIN_BOOTSTRAP='USB_A20'
   # Delays below are in seconds.
   _BOOTSTRAP_DELAY=0.1
-  _FIRST_FRAME_DELAY=0.1
-  _INTER_FRAME_DELAY=0.01
+  _FIRST_FRAME_DELAY=0.2
+  _INTER_FRAME_DELAY=0.02
```

Comment:
just curious: did you adjust these just to be on the safe side (no need to revert) or were these tight already and stopped working after the frame size update?

Reply:
These delays were indeed too small for the increased frame size. But I didn't experiment to reduce them to the minimum.